src/routes/safe/components/Apps/trackAppUsageCount.ts

```diff
@@ -0,0 +1,61 @@
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+import local from 'src/utils/storage/local'
+
+export const APPS_DASHBOARD = 'APPS_DASHBOARD'
+
+const TX_COUNT_WEIGHT = 2
+const OPEN_COUNT_WEIGHT = 1
+const MORE_RECENT_MULTIPLIER = 2
+const LESS_RECENT_MULTIPLIER = 1
+
+export type AppTrackData = {
+  [safeAppId: string]: {
+    timestamp: number
+    openCount: number
+    txCount: number
+  }
+}
+
+export const countOpen = (app: SafeApp): void => {
+  const trackData = local.getItem<AppTrackData>(APPS_DASHBOARD) || {}
+  const currentOpenCount = trackData[app.id]?.openCount || 0
+  const currentTxCount = trackData[app.id]?.txCount || 0
+  local.setItem(APPS_DASHBOARD, {
+    ...trackData,
+    [app.id]: {
+      ...trackData[app.id],
+      timestamp: Date.now(),
+      openCount: currentOpenCount + 1,
+      txCount: currentTxCount,
+    },
+  })
+}
+
+export const countTxs = (app: SafeApp): void => {
+  const trackData = local.getItem<AppTrackData>(APPS_DASHBOARD) || {}
+  const currentTxCount = trackData[app.id]?.txCount || 0
+  local.setItem(APPS_DASHBOARD, {
+    ...trackData,
+    [app.id]: { ...trackData[app.id], txCount: currentTxCount + 1 },
```

Comment:
There's duplicate logic here. You could use the 'base' object you create in `countTx()` in `countOpen()`.

I would also suggest more declaraitve names, e.g. `trackSafeAppOpenCount` and `trackSafeAppTxCount`